gmso/core/atom_type.py

```diff
@@ -19,44 +20,43 @@ class AtomType(Potential):
     non-bonded interaction between atoms or sites. The functional form of the
     potential is stored as a `sympy` expression and the parameters, with units,
     are stored explicitly.
-
-    Parameters
-    ----------
-    name : str, default="AtomType"
-        The name of the potential.
-    mass : unyt.unyt_quantity, optional, default=0.0 * unyt.g / u.mol
-        The mass of the atom type.
-    charge : unyt.unyt_quantity, optional, default=0.0 * unyt.elementary_charge
-        The charge of the atom type.
-    expression : str or sympy.Expr,
-                 default='4*epsilon*((sigma/r)**12 - (sigma/r)**6)',
-        The mathematical expression describing the functional form of the
-        potential describing this atom type, i.e. a Lennard-Jones potential.
-        The default value is a 12-6 Lennard-Jones potential.
-    parameters : dict of str : unyt.unyt_quantity pairs,
-        default={'sigma': 0.3 * u.nm, 'epsilon': 0.3 * u.Unit('kJ')},
-        The parameters of the potential describing this atom type and their
-        values, as unyt quantities.
-    independent_variables : str, sympy.Symbol, or list-like of str, sympy.Symbol
-        The independent variables of the functional form previously described.
-    atomclass : str, default=''
-        The class of the atomtype
-    doi : str
-        Digital Object Identifier of publication where this atom type was
-        specified
-    desc : str
-        Simple description of the atom type
-    overrides : set of str
-        Set of other atom types that this atom type overrides
-    definition : str
-        SMARTS string defining this atom type
-    topology: gmso.core.Topology, default=None
-        The topology of which this atom_type is a part of, default=None
-    set_ref: str
-        The string name of the bookkeeping set in gmso class.
-
     """
 
+    mass_: Optional[u.unyt_array] = Field(
+        0.0 * u.gram / u.mol,
+        description='The mass of the atom type'
+    )
+
+    charge_: Optional[u.unyt_array] = Field(
+        0.0 * u.elementary_charge,
+        description='The charge of the atom type'
+    )
+
```

Comment:
Should this contain an `element` attribute? There is an `element` field in the FF XMLs, but would this just be the name, or a string that can later be coerced into an element, etc.